### DISCUSSION TITLE
Display organization membership publicly

### DISCUSSION THREAD

**Discussion by Envek** [npm/feedback#418] [Website] [answered]
I created an organization recently but it displayed in [my profile's](https://www.npmjs.com/~envek) “Organizations” tab only when I am signed in and not displayed for other users (or in my browser's incognito mode). But I want to show that I belong to this organization as I'm proud to be a part of it.

Please, add ability to publish some organization's memebership. Maybe in profile settings (like it is done on GitHub, for example)

Thanks!

**Comment by MylesBorins** [comment, marked as answer]
Hey @Envek. Thanks for the suggestion. At this time we are not looking to make organization membership a public facing profile feature. There would be quite a bit of complexity both in UX and implementation in creating a system that would allow folks to choose what is public vs. what is private and it is not something that we would be prioritizing for engineering or design at this time
